.github/workflows/ci_tests.yaml

```
@@ -52,22 +52,15 @@ jobs:
       - name: Setup Miniconda
         uses: conda-incubator/setup-miniconda@v2.0.1
         with:
+          activate-environment: pygmt
           python-version: ${{ matrix.python-version }}
           channels: conda-forge
           miniconda-version: "latest"
 
       # Install GMT and other required dependencies from conda-forge
       - name: Install GMT and required dependencies
```

GMT isn't a special dependency.
```suggestion
      - name: Install dependencies
```